### PR TITLE
LibsBuilder: own LibsActivity via withOwnLibsActivityClass(Class)

### PR DESCRIPTION
**Pull request by Rainer-Lang**
- pass own LibsActivity via withOwnLibsActivityClass(Class)
- a bit Javadoc

### CODE REVIEW

File: library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java [hunk 1]
```diff
@@ -57,10 +58,23 @@
 
     public HashMap<String, HashMap<String, String>> libraryModification = null;
 
+    public Class ownLibsActivityClass = null;
```
Comment (mikepenz) [inline library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java]: Just set this to the `LibsActivity` by default. So you can remove the nullcheck on the bottom.

File: library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java [hunk 2]
```diff
@@ -57,10 +58,23 @@
...
     public LibsBuilder() {
     }
 
     /**
+     * Builder method to pass the an own LibsActivity.
+     *
+     * @param clazz Class
+     * @return this
+     */
+    public LibsBuilder withOwnLibsActivityClass(Class clazz) {
```
Comment (mikepenz) [inline library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java]: add annotation notnull

File: library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java [hunk 3]
```diff
@@ -501,7 +516,7 @@ public FastItemAdapter adapter(Context context) {
      * @return the intent to start the activity
      */
     public Intent intent(Context ctx) {
-        return intent(ctx, LibsActivity.class);
+        return intent(ctx, ownLibsActivityClass==null ? LibsActivity.class : ownLibsActivityClass);
```
Comment (mikepenz) [inline library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java]: remove nullcheck as proposed above

File: library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java (hunk 1 above)
Comment (Rainer-Lang) [inline library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java]: ok - will do

File: library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java (hunk 2 above)
Comment (Rainer-Lang) [inline library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java]: ok - will do

File: library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java (hunk 3 above)
Comment (Rainer-Lang) [inline library/src/main/java/com/mikepenz/aboutlibraries/LibsBuilder.java]: yes of course :)

**Comment by mikepenz** [comment]
Looks good :)
thanks

**Pull request merged**